app/payments.py

```diff
@@ -0,0 +1,114 @@
+from flask import Blueprint, render_template, session, jsonify, abort, url_for, redirect, flash, request
+from auth import login_required, flash_errors
+import functools
+from db import db, Teams, Players, Payments
+from passlib.hash import bcrypt_sha256
+import paypalrestsdk as paypal
+from paypalrestsdk import *
+import datetime
+import os
+
+payments_page = Blueprint('payment', __name__, url_prefix='/payment', template_folder='templates')
+
+def login_required(view):
+    @functools.wraps(view)
+    def wrapped_view(**kwargs):
+        # TODO: We should be doing check_token for this routine
+        if 'access_token' not in session:
+            flash("You must be logged in to see that!", "error")
+            return redirect(url_for('auth_page.signin'))
+
+        return view(**kwargs)
+
+    return wrapped_view
+
+paypal.configure({
+    "mode": "live",  # sandbox or live
+    "client_id": os.environ.get('client_id'),
```

Comment:
Change the os environment name that suggests it's for Paypal, and make it all caps. PAYPAL_CLIENTID and PAYPAL_SECRET works